### ISSUE TITLE
Tree states not working in Preview

### ISSUE THREAD

**Issue by jhpedemonte** [maqetta/maqetta#3138]
Running **Tests > Trees > Adding event handling to nodes** on Safari 6.

Previewed page after setting up events.  When I click or open any of the affected nodes, I get an error:

```
ReferenceError: Can't find variable: arguments
```

Refers to states inline script of `davinci.states.setState('myState',arguments[0])`, which is invoked by `dojo.eval` (which explains why there isn't an `arguments` var).

**Comment by aerwin** [comment]
Is this a zazl issue by chance?

**Comment by JonFerraiolo** [comment]
-> me. I put in the arguments[0] stuff.

**Comment by JonFerraiolo** [comment]
I'm going to get rid of the `arguments[0]`. It was an additional targeting the nested states feature, that we deferred out of M7. It looks like the 2nd parameter isn't going to fly with certain scenarios, which means I'll have to come up with a different algorithm for dealing with state change actions when there are nested states. Probably no big loss - I expect the revised algorithm will be as useful as the one I had implemented.
